packages/assets-controllers/src/TokenRatesController.test.ts

```diff
@@ -10,18 +9,19 @@ import {
 } from '@metamask/controller-utils';
 import type { InternalAccount } from '@metamask/keyring-api';
 import type {
+  NetworkClientConfiguration,
   NetworkClientId,
   NetworkState,
 } from '@metamask/network-controller';
 import { defaultState as defaultNetworkState } from '@metamask/network-controller';
-import type { NetworkClientConfiguration } from '@metamask/network-controller/src/types';
 import type { Hex } from '@metamask/utils';
 import { add0x } from '@metamask/utils';
 import assert from 'assert';
 import nock from 'nock';
 import { useFakeTimers } from 'sinon';
 
 import { advanceTime } from '../../../tests/helpers';
+import { createMockInternalAccount } from '../../accounts-controller/src/tests/mocks';
```

Comment:
With the `Node{16,Next}` option, subpath imports are no longer supported unless explicitly defined in the source package manifest's "exports" field. 

Here, they need to be replaced with relative paths.

Reply:
Ah I'm not sure how this slipped through the cracks! Makes sense.